app/src/main/contrib/actionTypes/ModellingGroupsTypes.ts

```diff
@@ -0,0 +1,14 @@
+import { ModellingGroupsState } from "../reducers/modellingGroupsReducer";
+import { ModellingGroup } from "../../shared/models/Generated";
+
+export enum TypeKeys {
+    GROUPS_FETCHED = "GROUPS_FETCHED",
+}
+
+export interface GroupsFetched {
+    type: TypeKeys.GROUPS_FETCHED;
```

Comment:
Could you leverage TypeScript here and just write:

```
export interface GroupsFetched {
    type: "GROUPS_FETCHED" | "SOME_OTHER_ACTION";
    data: ModellingGroup[]
}
```

Also, I'm not sure about the name "groups fetched". What about "received groups"?